src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -488,8 +513,8 @@ export default {
 				}
 			}
 
-			this.isLoading = false;
 			this.loan = result.data?.lend?.loan || null;
+			if (this.loan) this.isLoading = false;
```

Comment:
Is the component stuck in loading if `loan` is `null`? Is this a typical situation a user could encounter?

Reply:
this is to avoid a weird behavior when switching between default dummy data, for loading animations, and real data to display the loans.
only situation I think about this is to have a wrong data from api and stuck the component in loading state, but maybe is better than show a wrong loan.